### DISCUSSION TITLE
Brew fails to download binary for deno

### DISCUSSION THREAD

**Discussion by just-maiyak** [Homebrew/discussions#299] [Everyday usage] [answered]
I've been having this issue where, when installing/updating `deno`, brew builds from source because it cannot download the binary : 

```sh
>> brew install deno
(...)

==> Downloading https://homebrew.bintray.com/bottles/deno-1.6.0_1.catalina.bottle.tar.gz
#=#=#                                                                         
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "deno"
Download failed: https://homebrew.bintray.com/bottles/deno-1.6.0_1.catalina.bottle.tar.gz
Warning: Bottle installation failed: building from source.

(installing from source)
```
Does anyone have the same issue ?

**Comment by carlocab** [comment, marked as answer]
Sorry, I think this might've been caused by the Rust upgrade.

It should be fixed with the next version bump here: https://github.com/Homebrew/homebrew-core/pull/66920

**Reply by carlocab** [reply]
In the meantime, if you have Xcode installed (or don't mind installing it), you can build it from source using `brew install -s deno`.

**Reply by just-maiyak** [reply]
Building from source takes a long time, I have downloaded the binary from the github release for now.
Thank you for the quick response !

**Reply by carlocab** [reply]
This was fixed yesterday, by the way. You can now reinstall `deno` using Homebrew.
